packages/design-system/src/components/keyboard/index.js

```diff
@@ -59,7 +59,8 @@ const globalRef = createRef();
 
 function setGlobalRef() {
   if (!globalRef.current) {
-    globalRef.current = document.documentElement;
+    globalRef.current =
+      typeof document !== 'undefined' ? document.documentElement : null;
```

Comment:
Why not `globalThis.document?.documentElement || null`

Generally speaking, seems like this PR uses both different ways (`globalThis.document?.` vs `typeof document !== 'undefined'`). Can we settle on one?

Reply:
Yes, `globalThis.document?.documentElement || null` looks better here. At other places, I used `typeof document !== 'undefined'` because it looked more readable to write 

```js
typeof document !== 'undefined' ? document.getElementById(containerId) : null;`
```

as compared to 

```js
globalThis.document?.getElementById  ? document.getElementById(containerId) : null;`
```

Are you referring to any other instance?

Reply:
Updated all instances in https://github.com/GoogleForCreators/web-stories-wp/pull/11270/commits/438d0fa0537d351bae387dcbf77d54938f9dbdb8